### PR TITLE
Added Favicon, Missing SEO tags and Fixed a few CSS Issues

### PR DESCRIPTION
**Pull request by fazlerocks**
## New Additions
- Generated two different images
  - Favicon Image
  - Banner for social media shares
- Added meta tags for social media to all HTML pages. 
  - OG Tags
  - Twitter Tags
- Added SEO Meta description tag. 
## Fixes
- Changed title tag values in CoC.html, bangalore.html, los-angeles.html
- The CSS of `.brought-to-you` element was broken on larger displays — Fixed.
- Fixed image distortion in "You'll Learn to..." div. 

Cheers! :beers:

### CODE REVIEW

File: CoC.html [hunk 1]
```diff
@@ -2,7 +2,21 @@
 <html>
   <head>
     <link rel="stylesheet" href="styles/custom.css" />
-    <title>NodeTogether</title>
+    <title>Code of Conduct — NodeTogether</title>
+    <link rel="icon" type="image/png" href="/images/nodetogether-icon.png">
+    <!-- Tags for Social Media -->
+    <meta property="og:title" content="Code of Conduct — NodeTogether" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="http://www.nodetogether.org/events/2016/bangalore.html" />
```
Comment (bnb) [inline CoC.html:10]: This is the Bangalore URL, not the CoC URL. Needs to be changed, I assume?

File: events/2016/los-angeles.html [hunk 2]
```diff
@@ -3,6 +3,21 @@
   <head>
     <link rel="stylesheet" href="/styles/custom.css" />
     <title>NodeTogether 2016 - Los Angeles</title>
+    <link rel="icon" type="image/png" href="/images/nodetogether-icon.png">
+
+    <!-- Tags for Social Media -->
+    <meta property="og:title" content="NodeTogether in Los Angeles, CA, USA" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="http://www.nodetogether.org/events/2016/bangalore.html" />
```
Comment (bnb) [inline events/2016/los-angeles.html]: Change to `http://www.nodetogether.org/events/2016/los-angeles.html`

**Comment by bnb** [comment]
Did not check the `custom.css` edits - not sure what the best way to do it is beside adding them in the site's styles in my browser / some git Magic™ that I'm not aware of. 

OG and Twitter cards are definitely good to have. Should we add them now or wait until we have a slightly different system in place?

**Comment by ashleygwilliams** [comment]
@bnb you can use the command line instructions to pull this branch down and test it on your computer:
1. screen shot 2016-07-08 at 11 55 27 am
2. screen shot 2016-07-08 at 11 55 31 am
3. screen shot 2016-07-08 at 11 55 36 am

**Pull request closed without merging**